Added MK Point65 keyboards and keymaps

C44Supra opened the pull request:
## Description
Added the MK Point65 multi layout (6.25u and 7u) hotswap keyboard, and single layout hotswap RGB keyboard.

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Added keyboards
* Added default keymaps

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

C44Supra committed:
Added Point65 keyboards and keymaps

zvecr commented on keyboards/mechanicalkeyboards/point65/hotswap_rgb/config.h:
```diff
@@ -0,0 +1,44 @@
+ */
+
+#pragma once
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 16
+
+/* Keyboard Matrix Assignments
+#define MATRIX_ROW_PINS { GP24, GP23, GP22, GP21, GP25 }
+#define MATRIX_COL_PINS { GP28, GP27, GP26, GP29, GP0, GP11, GP10, GP9, GP8, GP7, GP6, GP5, GP4, GP3, GP2, GP1 }
+*/
+
+#define DEBOUNCE 5
+
```
```suggestion

```

zvecr commented on keyboards/mechanicalkeyboards/point65/hotswap_rgb/config.h:
```diff
@@ -0,0 +1,44 @@
+/* Double tap the side button to enter bootloader */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+#ifdef RGB_MATRIX_ENABLE
+    #define WS2812_PIO_USE_PIO1
+
+    /* Enable Framebuffer and keypress effects */
+    #define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+    #define RGB_MATRIX_KEYPRESSES
+#endif
```
```suggestion
```

zvecr commented on keyboards/mechanicalkeyboards/point65/hotswap_rgb/keyboard.json:
```diff
@@ -0,0 +1,232 @@
+            "solid_multisplash":            true
+        },
+        "default": {
+            "animation":    "cycle_left_right",
+            "on":           true,
```
```suggestion
```

zvecr commented on keyboards/mechanicalkeyboards/point65/hotswap/keyboard.json:
```diff
@@ -0,0 +1,314 @@
+        "default": {
+            "animation":    "cycle_left_right",
+            "on":           true,
+            "val":          192
```
Default is greater than max?

zvecr commented on keyboards/mechanicalkeyboards/point65/hotswap/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,79 @@
+    // end layer function
+
+    // capslock indicator
+    if (!rgb_matrix_indicators_user()) {
+        return false;
+    }
+
```
```suggestion
```

dunk2k reviewed (changes requested):
1. Remove layout `LAYOUT_625u`, from hotswap/keyboard.json, as it is identical to `LAYOUT_all` (i.e. needless data duplication)
2. Rename layout names to match Community Layouts, as per PR Checklist
  1. point65/hotswap: layout alias added for `LAYOUT_all`
  2. community layout support added
  3. `readme.md` instructions adjusted
3. `info.json` removed

For these changes to not cause issue(s), __if__ they are accepted, the following directories and their contents are to be deleted:
- keyboards/mechanicalkeyboards/point65/hotswap/keymaps/default_625u

dunk2k commented on keyboards/mechanicalkeyboards/point65/readme.md:
```diff
@@ -0,0 +1,33 @@
+# Mechanical Keyboards Point65
+
+![Mechanical Keyboards Point65](https://mechanicalkeyboards.com/cdn/shop/files/17705-MIJ95-Point-65-Keyboard-Kit.jpg)
```
could this image be re-hosted on imgur?

C44Supra committed:
Made requested changes as per suggestions on qmk PR #25392

- rehosted readme image
- updated readme build commands
- switched to community layouts
- removed redundant default_625 keymap folder
- removed info.json
- removed redundant / obsolete / superfluous rgb matrix definitions, debounce, and matrix definitions from config files
- updated keyboard.json files to better reflect individual boards
- changed default brightness value in hotswap rgb keyboard.json, added it to hotswap keyboard.json
- fixed issue with RGB matrix definition in hotswap rgb keyboard.json
- removed redundant rgb_matrix_indicators_user function from keymaps

C44Supra commented on keyboards/mechanicalkeyboards/point65/hotswap/keyboard.json:
```diff
@@ -0,0 +1,314 @@
+                {   "matrix": [4, 15],  "label":"KC_RGHT",              "x":15,     "y":4,      "w":1       }
+            ]
+        },
+        "LAYOUT_7u": {
```
Noted, updated, and committed =)

C44Supra commented on keyboards/mechanicalkeyboards/point65/hotswap_rgb/keyboard.json:
```diff
@@ -0,0 +1,232 @@
+    },
+    "diode_direction": "COL2ROW",
+    "layouts": {
+        "LAYOUT": {
```
Just to clarify, the fact that you're not suggesting I use the layouts_alias var here and change LAYOUT to LAYOUT_65_ansi_blocker, is because there's only one layout macro defined for this board, correct?

dunk2k replied:
> Just to clarify, the fact that you're not suggesting I use the layouts_alias var here and change LAYOUT to LAYOUT_65_ansi_blocker, is because there's only one layout macro defined for this board, correct?

That's correct :)

C44Supra replied:
Understood! Thank you =)

C44Supra committed:
Addendum to changes requested on qmk pr #25392

Removed redundant default animation and state variables for both PCBs

C44Supra commented on keyboards/mechanicalkeyboards/point65/hotswap/keyboard.json:
```diff
@@ -0,0 +1,314 @@
+        },
+        "default": {
+            "animation":    "cycle_left_right",
+            "on":           true,
```
Dangit, that one I was unaware of. Changed as requested.

waffle87 reviewed (changes requested):
If you could consolidate shared configuration between the `hotswap` and `hotswap_rgb` revisions into a top-level `info.json` file, and hardware-specific configuration in the individual `keyboard.json` files.

C44Supra committed:
Made additional changes requested on qmk #25392

- consolidated keyboard.json common values into info.json
- re-formatted all three json files
- updated keymap

C44Supra committed:
Updated RGB Matrix keycodes in hotswap_rgb keyboard default keymap

C44Supra committed:
info.json and keyboard.json updates

- moved matrix_pins definitions back into individual keyboard.json files
- removed rgb matrix from info.json

Moving matrix_pins definition to info.json renders the rgb matrix of hotswap_rgb inoperable, even though the definitions themselves are identical.

dunk2k commented on keyboards/mechanicalkeyboards/point65/info.json:
```diff
@@ -0,0 +1,72 @@
+    "manufacturer": "Mechanical Keyboards",
+    "maintainer": "C44Supra",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
```
moved from keyboard.json
```suggestion
    "diode_direction": "COL2ROW",
    "matrix_pins": {
        "cols": ["GP28", "GP27", "GP26", "GP29", "GP0", "GP11", "GP10", "GP9", "GP8", "GP7", "GP6", "GP5", "GP4", "GP3", "GP2", "GP1"]
    },
```

dunk2k commented on keyboards/mechanicalkeyboards/point65/hotswap_rgb/config.h:
Please move this file to parent folder /keyboards/mechanicalkeyboards/point65/

dunk2k commented on keyboards/mechanicalkeyboards/point65/hotswap_rgb/keyboard.json:
```diff
@@ -0,0 +1,159 @@
+                {"label": "KC_CAPS", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
+                {"label": "KC_A", "matrix": [2, 1], "x": 1.75, "y": 2, "w": 1},
+                {"label": "KC_S", "matrix": [2, 2], "x": 2.75, "y": 2, "w": 1},
+                {"label": "KC_D", "matrix": [2, 3], "x": 3.75, "y": 2, "w": 1},
+                {"label": "KC_F", "matrix": [2, 4], "x": 4.75, "y": 2, "w": 1},
+                {"label": "KC_G", "matrix": [2, 5], "x": 5.75, "y": 2, "w": 1},
+                {"label": "KC_H", "matrix": [2, 6], "x": 6.75, "y": 2, "w": 1},
+                {"label": "KC_J", "matrix": [2, 7], "x": 7.75, "y": 2, "w": 1},
+                {"label": "KC_K", "matrix": [2, 8], "x": 8.75, "y": 2, "w": 1},
+                {"label": "KC_L", "matrix": [2, 9], "x": 9.75, "y": 2, "w": 1},
+                {"label": "KC_SCLN", "matrix": [2, 10], "x": 10.75, "y": 2, "w": 1},
+                {"label": "KC_QUOT", "matrix": [2, 11], "x": 11.75, "y": 2, "w": 1},
+                {"label": "KC_ENTER", "matrix": [2, 12], "x": 12.75, "y": 2, "w": 2.25},
+                {"label": "KC_PGDN", "matrix": [2, 15], "x": 15, "y": 2, "w": 1},
+                {"label": "KC_LSFT", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
+                {"label": "KC_Z", "matrix": [3, 1], "x": 2.25, "y": 3, "w": 1},
+                {"label": "KC_X", "matrix": [3, 2], "x": 3.25, "y": 3, "w": 1},
+                {"label": "KC_C", "matrix": [3, 3], "x": 4.25, "y": 3, "w": 1},
+                {"label": "KC_V", "matrix": [3, 4], "x": 5.25, "y": 3, "w": 1},
+                {"label": "KC_B", "matrix": [3, 5], "x": 6.25, "y": 3, "w": 1},
+                {"label": "KC_N", "matrix": [3, 6], "x": 7.25, "y": 3, "w": 1},
+                {"label": "KC_M", "matrix": [3, 7], "x": 8.25, "y": 3, "w": 1},
+                {"label": "KC_COMM", "matrix": [3, 8], "x": 9.25, "y": 3, "w": 1},
+                {"label": "KC_DOT", "matrix": [3, 9], "x": 10.25, "y": 3, "w": 1},
+                {"label": "KC_SLSH", "matrix": [3, 10], "x": 11.25, "y": 3, "w": 1},
+                {"label": "KC_RSFT", "matrix": [3, 11], "x": 12.25, "y": 3, "w": 1.75},
+                {"label": "KC_UP", "matrix": [3, 14], "x": 14, "y": 3, "w": 1},
+                {"label": "KC_END", "matrix": [3, 15], "x": 15, "y": 3, "w": 1},
+                {"label": "KC_LCTL", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
+                {"label": "KC_LGUI", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"label": "KC_LALT", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "KC_SPC", "matrix": [4, 5], "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "KC_RALT", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
+                {"label": "_FN", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "KC_LEFT", "matrix": [4, 13], "x": 13, "y": 4, "w": 1},
+                {"label": "KC_DOWN", "matrix": [4, 14], "x": 14, "y": 4, "w": 1},
+                {"label": "KC_RGHT", "matrix": [4, 15], "x": 15, "y": 4, "w": 1}
+            ]
+        }
+    }
```
move to info.json
```suggestion
```

dunk2k commented on keyboards/mechanicalkeyboards/point65/hotswap/keyboard.json:
```diff
@@ -0,0 +1,167 @@
+                {"label": "KC_CAPS", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
+                {"label": "KC_A", "matrix": [2, 1], "x": 1.75, "y": 2, "w": 1},
+                {"label": "KC_S", "matrix": [2, 2], "x": 2.75, "y": 2, "w": 1},
+                {"label": "KC_D", "matrix": [2, 3], "x": 3.75, "y": 2, "w": 1},
+                {"label": "KC_F", "matrix": [2, 4], "x": 4.75, "y": 2, "w": 1},
+                {"label": "KC_G", "matrix": [2, 5], "x": 5.75, "y": 2, "w": 1},
+                {"label": "KC_H", "matrix": [2, 6], "x": 6.75, "y": 2, "w": 1},
+                {"label": "KC_J", "matrix": [2, 7], "x": 7.75, "y": 2, "w": 1},
+                {"label": "KC_K", "matrix": [2, 8], "x": 8.75, "y": 2, "w": 1},
+                {"label": "KC_L", "matrix": [2, 9], "x": 9.75, "y": 2, "w": 1},
+                {"label": "KC_SCLN", "matrix": [2, 10], "x": 10.75, "y": 2, "w": 1},
+                {"label": "KC_QUOT", "matrix": [2, 11], "x": 11.75, "y": 2, "w": 1},
+                {"label": "KC_ENTER", "matrix": [2, 12], "x": 12.75, "y": 2, "w": 2.25},
+                {"label": "KC_PGDN", "matrix": [2, 15], "x": 15, "y": 2, "w": 1},
+                {"label": "KC_LSFT", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
+                {"label": "KC_Z", "matrix": [3, 1], "x": 2.25, "y": 3, "w": 1},
+                {"label": "KC_X", "matrix": [3, 2], "x": 3.25, "y": 3, "w": 1},
+                {"label": "KC_C", "matrix": [3, 3], "x": 4.25, "y": 3, "w": 1},
+                {"label": "KC_V", "matrix": [3, 4], "x": 5.25, "y": 3, "w": 1},
+                {"label": "KC_B", "matrix": [3, 5], "x": 6.25, "y": 3, "w": 1},
+                {"label": "KC_N", "matrix": [3, 6], "x": 7.25, "y": 3, "w": 1},
+                {"label": "KC_M", "matrix": [3, 7], "x": 8.25, "y": 3, "w": 1},
+                {"label": "KC_COMM", "matrix": [3, 8], "x": 9.25, "y": 3, "w": 1},
+                {"label": "KC_DOT", "matrix": [3, 9], "x": 10.25, "y": 3, "w": 1},
+                {"label": "KC_SLSH", "matrix": [3, 10], "x": 11.25, "y": 3, "w": 1},
+                {"label": "KC_RSFT", "matrix": [3, 11], "x": 12.25, "y": 3, "w": 1.75},
+                {"label": "KC_UP", "matrix": [3, 14], "x": 14, "y": 3, "w": 1},
+                {"label": "KC_END", "matrix": [3, 15], "x": 15, "y": 3, "w": 1},
+                {"label": "KC_LCTL", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
+                {"label": "KC_LGUI", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"label": "KC_LALT", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "KC_SPC", "matrix": [4, 5], "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "KC_RALT", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
+                {"label": "_FN", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "KC_LEFT", "matrix": [4, 13], "x": 13, "y": 4, "w": 1},
+                {"label": "KC_DOWN", "matrix": [4, 14], "x": 14, "y": 4, "w": 1},
+                {"label": "KC_RGHT", "matrix": [4, 15], "x": 15, "y": 4, "w": 1}
+            ]
+        }
+    }
```
move to info.json, as LAYOUT_65_ansi_blocker
```suggestion
        }
```

dunk2k commented on keyboards/mechanicalkeyboards/point65/hotswap/keyboard.json:
```diff
@@ -0,0 +1,167 @@
+    },
+    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_tsangan"],
+    "layout_aliases": {
+        "LAYOUT_65_ansi_blocker": "LAYOUT_all"
```
```suggestion
        "LAYOUT_all": "LAYOUT_65_ansi_blocker"
```

dunk2k commented on keyboards/mechanicalkeyboards/point65/info.json:
```diff
@@ -0,0 +1,72 @@
+    },
+    "ws2812": {
+        "driver": "vendor"
+    }
```
```suggestion
    },
    "layouts": {
        "LAYOUT_65_ansi_blocker": {
            "layout": [
                {"label": "KC_ESC", "matrix": [0, 0], "x": 0, "y": 0, "w": 1},
                {"label": "KC_1", "matrix": [0, 1], "x": 1, "y": 0, "w": 1},
                {"label": "KC_2", "matrix": [0, 2], "x": 2, "y": 0, "w": 1},
                {"label": "KC_3", "matrix": [0, 3], "x": 3, "y": 0, "w": 1},
                {"label": "KC_4", "matrix": [0, 4], "x": 4, "y": 0, "w": 1},
                {"label": "KC_5", "matrix": [0, 5], "x": 5, "y": 0, "w": 1},
                {"label": "KC_6", "matrix": [0, 6], "x": 6, "y": 0, "w": 1},
                {"label": "KC_7", "matrix": [0, 7], "x": 7, "y": 0, "w": 1},
                {"label": "KC_8", "matrix": [0, 8], "x": 8, "y": 0, "w": 1},
                {"label": "KC_9", "matrix": [0, 9], "x": 9, "y": 0, "w": 1},
                {"label": "KC_0", "matrix": [0, 10], "x": 10, "y": 0, "w": 1},
                {"label": "KC_MINUS", "matrix": [0, 11], "x": 11, "y": 0, "w": 1},
                {"label": "KC_EQUAL", "matrix": [0, 12], "x": 12, "y": 0, "w": 1},
                {"label": "KC_BSPC", "matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"label": "KC_HOME", "matrix": [0, 15], "x": 15, "y": 0, "w": 1},
                {"label": "KC_TAB", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "KC_Q", "matrix": [1, 1], "x": 1.5, "y": 1, "w": 1},
                {"label": "KC_W", "matrix": [1, 2], "x": 2.5, "y": 1, "w": 1},
                {"label": "KC_E", "matrix": [1, 3], "x": 3.5, "y": 1, "w": 1},
                {"label": "KC_R", "matrix": [1, 4], "x": 4.5, "y": 1, "w": 1},
                {"label": "KC_T", "matrix": [1, 5], "x": 5.5, "y": 1, "w": 1},
                {"label": "KC_Y", "matrix": [1, 6], "x": 6.5, "y": 1, "w": 1},
                {"label": "KC_U", "matrix": [1, 7], "x": 7.5, "y": 1, "w": 1},
                {"label": "KC_I", "matrix": [1, 8], "x": 8.5, "y": 1, "w": 1},
                {"label": "KC_O", "matrix": [1, 9], "x": 9.5, "y": 1, "w": 1},
                {"label": "KC_P", "matrix": [1, 10], "x": 10.5, "y": 1, "w": 1},
                {"label": "KC_LEFT_BRACKET", "matrix": [1, 11], "x": 11.5, "y": 1, "w": 1},
                {"label": "KC_RIGHT_BRACKET", "matrix": [1, 12], "x": 12.5, "y": 1, "w": 1},
                {"label": "KC_BACKSLASH", "matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"label": "KC_PGUP", "matrix": [1, 15], "x": 15, "y": 1, "w": 1},
                {"label": "KC_CAPS", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "KC_A", "matrix": [2, 1], "x": 1.75, "y": 2, "w": 1},
                {"label": "KC_S", "matrix": [2, 2], "x": 2.75, "y": 2, "w": 1},
                {"label": "KC_D", "matrix": [2, 3], "x": 3.75, "y": 2, "w": 1},
                {"label": "KC_F", "matrix": [2, 4], "x": 4.75, "y": 2, "w": 1},
                {"label": "KC_G", "matrix": [2, 5], "x": 5.75, "y": 2, "w": 1},
                {"label": "KC_H", "matrix": [2, 6], "x": 6.75, "y": 2, "w": 1},
                {"label": "KC_J", "matrix": [2, 7], "x": 7.75, "y": 2, "w": 1},
                {"label": "KC_K", "matrix": [2, 8], "x": 8.75, "y": 2, "w": 1},
                {"label": "KC_L", "matrix": [2, 9], "x": 9.75, "y": 2, "w": 1},
                {"label": "KC_SCLN", "matrix": [2, 10], "x": 10.75, "y": 2, "w": 1},
                {"label": "KC_QUOT", "matrix": [2, 11], "x": 11.75, "y": 2, "w": 1},
                {"label": "KC_ENTER", "matrix": [2, 12], "x": 12.75, "y": 2, "w": 2.25},
                {"label": "KC_PGDN", "matrix": [2, 15], "x": 15, "y": 2, "w": 1},
                {"label": "KC_LSFT", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"label": "KC_Z", "matrix": [3, 1], "x": 2.25, "y": 3, "w": 1},
                {"label": "KC_X", "matrix": [3, 2], "x": 3.25, "y": 3, "w": 1},
                {"label": "KC_C", "matrix": [3, 3], "x": 4.25, "y": 3, "w": 1},
                {"label": "KC_V", "matrix": [3, 4], "x": 5.25, "y": 3, "w": 1},
                {"label": "KC_B", "matrix": [3, 5], "x": 6.25, "y": 3, "w": 1},
                {"label": "KC_N", "matrix": [3, 6], "x": 7.25, "y": 3, "w": 1},
                {"label": "KC_M", "matrix": [3, 7], "x": 8.25, "y": 3, "w": 1},
                {"label": "KC_COMM", "matrix": [3, 8], "x": 9.25, "y": 3, "w": 1},
                {"label": "KC_DOT", "matrix": [3, 9], "x": 10.25, "y": 3, "w": 1},
                {"label": "KC_SLSH", "matrix": [3, 10], "x": 11.25, "y": 3, "w": 1},
                {"label": "KC_RSFT", "matrix": [3, 11], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "KC_UP", "matrix": [3, 14], "x": 14, "y": 3, "w": 1},
                {"label": "KC_END", "matrix": [3, 15], "x": 15, "y": 3, "w": 1},
                {"label": "KC_LCTL", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"label": "KC_LGUI", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"label": "KC_LALT", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"label": "KC_SPC", "matrix": [4, 5], "x": 3.75, "y": 4, "w": 6.25},
                {"label": "KC_RALT", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
                {"label": "_FN", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
                {"label": "KC_LEFT", "matrix": [4, 13], "x": 13, "y": 4, "w": 1},
                {"label": "KC_DOWN", "matrix": [4, 14], "x": 14, "y": 4, "w": 1},
                {"label": "KC_RGHT", "matrix": [4, 15], "x": 15, "y": 4, "w": 1}
            ]
        }
    }
```

dunk2k reviewed (changes requested):
suggestions to deduplicate data

C44Supra committed:
Further file consolidations as requested

C44Supra committed:
json re-formatted w/ format-json

C44Supra committed:
Added RGB Matrix controls to Hotswap keymaps as requested

fauxpark commented on keyboards/mechanicalkeyboards/point65/readme.md:
```diff
@@ -0,0 +1,33 @@
+## Description
+A 65% keyboard by [Mechanical Keyboards](https://mechanicalkeyboards.com) available as either multi layout hotswap, or hotswap with RGB per key backlight.
+
+> Keyboard Maintainer: [C44Supra](https://github.com/C44Supra)
+>
+> Hardware Supported: MK Point65 hotswap, MK Point65 hotswap RGB
+>
+> Hardware Availability: [MK Point65 Keyboard Kit](https://mechanicalkeyboards.com/products/mk-point65-keyboard-kit)
```
```suggestion
* Keyboard Maintainer: [C44Supra](https://github.com/C44Supra)
* Hardware Supported: MK Point65 hotswap, MK Point65 hotswap RGB
* Hardware Availability: [MK Point65 Keyboard Kit](https://mechanicalkeyboards.com/products/mk-point65-keyboard-kit)
```

zvecr replied:
Note the suggestion here has `*` and not `>`, d8149da4c389be30fc9c293c79b3f5f14d9fda93 does not implement this suggestion correctly.

C44Supra replied:
Fixed!

zvecr commented on keyboards/mechanicalkeyboards/point65/info.json:
```diff
@@ -0,0 +1,148 @@
+            "splash": true,
+            "typing_heatmap": true
+        },
+        "default": {
+            "val": 128
+        },
```
Default would already be `max_brightness`
```suggestion
```

C44Supra committed:
Removed default RGB brightness value setting, reformatted readme.md

C44Supra committed:
fixed oversight in list

C44Supra commented:
All of the requested changes have been made. If there's anything else I'm missing, please let me know?

C44Supra committed:
Removed default key widths

C44Supra commented on keyboards/mechanicalkeyboards/point65/info.json:
```diff
@@ -0,0 +1,145 @@
+    "layouts": {
+        "LAYOUT_65_ansi_blocker": {
+            "layout": [
+                {"label": "KC_ESC", "matrix": [0, 0], "x": 0, "y": 0, "w": 1},
```
Done.

C44Supra committed:
Moved rgb matrix indicator functions to keyboard level

C44Supra committed:
Minor variable name correction

fauxpark commented on keyboards/mechanicalkeyboards/point65/info.json:
```diff
@@ -0,0 +1,145 @@
+    "layouts": {
+        "LAYOUT_65_ansi_blocker": {
+            "layout": [
+                {"label": "KC_ESC", "matrix": [0, 0], "x": 0, "y": 0},
```
Please remove the labels or change them to not be the keycodes, as that may confuse people. Key assigments are done in the keymap, and IMO the layout definition should not be opinionated about it.

C44Supra replied:
Removed as requested.

fauxpark commented on keyboards/mechanicalkeyboards/point65/hotswap/keyboard.json:
```diff
@@ -0,0 +1,95 @@
+    "layouts": {
+        "LAYOUT_65_ansi_blocker_tsangan": {
+            "layout": [
+                {"label": "KC_ESC", "matrix": [0, 0], "x": 0, "y": 0},
```
As above.

C44Supra committed:
Removed label variables from layout definitions

zvecr reviewed (changes requested):
* `_kb` functions not calling `_user`
* Using deprecated `RGB` and `HSV` types

C44Supra committed:
Merge branch 'qmk:master' into beanboards/point65

C44Supra committed:
Added _user calls, updated HSV and RGB types, added missing end-of-file newlines.
